src/rapids_dependency_file_generator/cli.py

```diff
@@ -35,10 +37,17 @@ def validate_args(argv):
 
     codependent_args = parser.add_argument_group("optional, but codependent")
     codependent_args.add_argument(
-        "--file_key",
+        "-f",
```

Comment:
Can we remove the short form `-f`? I like forcing callers to spell out `--file-key` to make it clear for readers who aren't familiar with DFG. This would also avoid confusion with `-f` meaning `--force` (this is a common point of confusion in conda which mixes `--file` and `--force`, both abbreviated as `-f` depending on the command).
```suggestion
```